custom_model_runner/requirements.txt

```diff
@@ -25,3 +25,4 @@ julia<=0.5.7
 termcolor
 packaging
 markupsafe<=2.1.3
+pydantic==2.9.2
```

Comment:
I think we should consider bumping DRUM version mainly because of new `pydantic` dependency

Reply:
Yes, when I publish a new DRUM release, I'll bump the version. The next version will likely include more changes than this one.